ReactAndroid/src/main/java/com/facebook/react/modules/deviceinfo/DeviceInfoModule.java

```diff
@@ -9,11 +9,6 @@
 
 package com.facebook.react.modules.deviceinfo;
 
-import javax.annotation.Nullable;
```

Comment:
Unless there's any styling rule requirement, moving of the imports is most likely unnecessary.

Reply:
Sure, that was Android Studio moving them around automatically but i can set them back.